tests/scripts/run-throughput-perf.py

```diff
@@ -165,6 +168,7 @@ def validate_arg(arg, check):
     valid_run_types = ['rolling', 'private']
     valid_os = ['Windows_NT', 'Ubuntu14.04']
     valid_opt_levels = ['full_opt', 'min_opt']
+    valid_jit_names = ['ryujit', 'legacy_backend']
```

Comment:
Since legacy_backend is only valid for x86, I'd like to see this changed to resemble the valid_arches list so:

`valid_jit_names = {'x64': ['ryujit'], 'x86': ['ryujit', 'legacy_backend']}`